src/js/timeseries.js

```diff
@@ -6,7 +6,8 @@ import { el, prettyDate, chartExportOptions, drawMetricSummary } from './utils';
 
 
 function timeseries(metric, options, start, end) {
-	const dataUrl = `https://cdn.httparchive.org/reports/${metric}.json`;
+	// TODO: remove cache buster
```

Comment:
Roll back unneeded changes to this file.

Reply:
This is needed temporarily while the CDN-hosted reports aren't stable. Cloud Storage has quite some aggressive caching in place. It's marked as `TODO` to be removed.